crates/bytecode/src/tests.rs

```diff
@@ -0,0 +1,183 @@
+#[cfg(test)]
```

Comment:
rm `src/tests.rs` and move this code inside `iterator.rs` it is better practice to have it in ona place